addons/pos_event/static/src/app/components/popup/event_configurator_popup/event_configurator_popup.js

```diff
@@ -47,19 +47,40 @@ export class EventConfiguratorPopup extends Component {
             return this.props.availabilityPerTicket[ticket.id];
         }
         const ticketAvailability = this.props.availabilityPerTicket[ticket.id][this.slotId];
-        const existingUnsyncRegistration = this.pos.models["event.registration"].filter(
-            (r) => typeof r.id === "string" && r.event_slot_id.id === this.slotId
-        );
```

Comment:
If we make several bookings before paying, this was including the 'pending' tickets in the tickets 'unavailable' I believe, is it just dead code / useless?

Reply:
( As you modify availabilityPerTicket accordingly, should be good! )